docker-compose-local.yml

```diff
@@ -0,0 +1,34 @@
+# For local development, only database and redis is running
+#
+# docker compose -f docker-compose-local.yml up -d
+#
+
+services:
```

Comment:
I wanted to avoid having two docker-compose files to avoid unnecessary duplication. I think that we don't need it, because we can start just selected services from the main compose file, right? E.g. locally, the user can just run `docker-compose up [SERVICE1] [SERVICE2]`. Or even better, we could use [profiles](https://docs.docker.com/compose/how-tos/profiles/), and assign a "local" profile only to Postgre + Redis, so that these two can be started easily during development, without the rest.

Reply:
Okay, no problem! It was just from experience that I’ve noticed people don’t read the documentation and start the Docker Compose without hesitation :D

Reply:
Well, good point :) But in that case I don't think that people could even conjure the correct `docker-compose -f <XYZ>` command and find the right compose file, so :) And we have like 3 people contributing to Kelvin anyway, so this is not a big concern :D

Reply:
Done, created profile "prod", local development is still just `docker compose up -d`